### PR TITLE
Fixed bin/rlp CLI command bug

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

**Comment by holgerd77** [comment]
Have for now replaced the usage of ``util.promisify()`` in the new CLI command integration test with the callback version, since this was breaking on Node 6. However this raises the general question, if setup generally can be improved to also transpile tests and then run these transpiled tests so that we can independently from the targeted Node distribution version can use modern JS in the test setup.

File: test/integration.spec.ts [hunk 1]
```diff
@@ -9,16 +10,10 @@ describe('Distribution:', function() {
...
 })
 
-/*
 describe('CLI command:', function() {
   it('should be able to run CLI command', function() {
-    const execP = util.promisify(exec)
-
-    async function bin() {
-      const { stdout, stderr } = await execP('./bin/rlp encode "[ 5 ]"')
+    exec('./bin/rlp encode "[ 5 ]"', (_error, stdout, _stderr) => {
```
Comment (holgerd77) [inline test/integration.spec.ts]: Not sure, I was running into linting errors like ``'error' is declared but its value is never read`` here, however I didn't want to explicitly handle error and stuff here, so I googled towards the underscore solution. Is this a good way to do it?

Comment (krzkaczor) [reply test/integration.spec.ts]: Yup, I do this like this myself. Furthermore: `_stderr` could be deleted entirely since it's the last parameter.

**Comment by krzkaczor** [comment]
>  However this raises the general question, if setup generally can be improved to also transpile tests and then run these transpiled tests so that we can independently from the targeted Node distribution version can use modern JS in the test setup. 

Yes we can! (but that woudln't work anyway because we will transpile language features not node API). But bascially we can create tsconfig (like `tsconfig.test.json` targeting whatever node 6 supports and use to exectute with mocha (using [TS_NODE_PROJECT](https://github.com/TypeStrong/ts-node#cli-and-programmatic-options) ). 

Buuuut when I think about it we already did language transpilation since we target ES5 and tests failed due to different node API. We could change `lib` (and possible `@types/node` version) settings to find these mistakes automatically.

**Comment by holgerd77** [comment]
Ah, makes very much sense, for the moment maybe no need to do imminent changes in the setup.

Thanks for approval!

**Pull request merged**